src/km.js

```diff
@@ -299,7 +299,10 @@
     ZMI() { D.prf.zoom(Math.min(12, D.prf.zoom() + 1)); },
     ZMO() { D.prf.zoom(Math.max(-10, D.prf.zoom() - 1)); },
     ZMR() { D.prf.zoom(0); },
-
+    RLB() {
+      D.db.setItem('lbarOrder', D.lb.order);
```

Comment:
I noticed in the debugger that `D.lb.order` always seem to have the default order, so why not just reuse it and set it in DB so the reset-action is persistent. That was my thinking anyways

Reply:
The preferences `D.prf` use `D.db` as a storage module, so we don't access it directly.
The properties of `D.prf` handle the storage and retrieval of values and can be used to register listeners for changes to them (see the first few lines of [prf.js](https://github.com/Dyalog/ride/blob/master/src/prf.js)
In theory you can reset a property to default values with:

`D.prf.mySetting(D.prf.mySetting.getDefault())`

This might not work in this case because `D.lb.order` is an object that gets replaced when connecting to an interpreter and receiving a list of supported glyphs (so the `D.prf.lbarOrder.getDefault()` may return the original object rather than the new) . But you should be able to do:

`D.prf.lbarOrder(D.lb.order)`

Reply:
Done! Also, thanks for the explanation of the code. I must admit, for my feeble brain, it was a bit of a mouthful finding my around. I appreciate the added context.